content/docs/reference/config/installation-options-changes/index.md

```diff
@@ -17,443 +17,217 @@ The tables below show changes made to the installation options used to customize
 
 ## Modified configuration options
 
-### Modified `servicegraph` key/value pairs
+### Modified `kiali` key/value pairs
 
 | Key | Old Default Value | New Default Value | Old Description | New Description |
 | --- | --- | --- | --- | --- |
-| `servicegraph.ingress.hosts` | `servicegraph.local` | `servicegraph.local` |  | `Used to create an Ingress record.` |
+| `kiali.hub` | `docker.io/kiali` | `quay.io/kiali` |  |  |
```

Comment:
@rcaballeromx We parse the values.yaml file to get the description to populate the Description field. If there are no descriptions, we leave this field empty. There are also some values.yaml file where some of the configuration options are commented out. Since we do not want those to be included in the descriptions of the following configuration option, we have filters in place to see if a comment field has a format: <something>: <something>, then we ignore those descriptions (to be careful that we do not include comments that might contain commented out options.) So these values.yaml files need some cleanup but we refrained from doing that since @ostromart is working on developing a cleaner approach to get the schema of these options which also includes providing descriptions for these options.

Reply:
@johnma14 could you file an issue referencing this PR for us to follow up after release?
You can base it on this example #4478